### DISCUSSION TITLE
🚀 Feature Request: Event Notifications in `wrangler.toml`

### DISCUSSION THREAD

**Discussion by helloimalastair** [cloudflare/workers-sdk#13490] [Feature Requests]
### Describe the solution

It would be nice to be able to be able to use `wrangler.toml`/`.json` as a source-of-truth for which Buckets/Queues generate/handle Event Notifications.

Configuration should *probably* be in Queue Consumer Worker, though not sure what to do if a manually connected Event Notification isn't explicitly defined in `wrangler.toml`. Should it be removed? Should it be left out?

**Comment by andyjessop** [comment]
@irvinebroque @jonesphillip looping you guys in in case this is of interest.
